.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,3 +16,20 @@ Bullet points like
 really help with making this more readable.
 
 Fixes/Closes/Related-to #1 (enter issue number, except in rare cases where none exists).
+
+It is also helpful to add an emoji before the commit message to identify the kind of work done on a single commit. See the following suggestions below:
+
+- :bug: - (bug) bug fix
+- :sparkles: - (sparkles) new feature
+- :exclamation: - (exclamation) tests
+- :fire: - (fire) P0
+- :rocket: - (rocket) performance improvements
+- :lipstick: - (lipstick) css/styling
+- :wheelchair: - (wheelchair) accessability
```

Comment:
The wheelchair is nice because `:acc` autocompletes to it, but it'd be nice if there was something a little more inclusive of all people with accessibility needs rather than only those with physical disability.

Reply:
any suggestions? I was thinking maybe of 🅰️ ?

Reply:
:a: matches but it's a little plain. :busts_in_silhouette: could be good? Maybe we could call the category usability and accessibility? I'm also ok with :wheelchair:, it's more a critique of GitHub that :accessibility only autosuggests just the wheelchair image.